src/Student9/Ex4/MinMaxCounter.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react'
+
+export const MinMaxCounter = () => {
+    let [count, setCount] = useState(0);
+    let [Min,setMin] = useState(0)
+    let[max,setMax] = useState(0)
+    let increase = () => {
+        setCount(count + 1);
+    }
+
+    let decrease = () => {
+        setCount(count - 1)
+    }
+    return(
+        <div>
+        <button onClick={decrease}>Down</button>
+        {count}
+        <button onClick={increase}>Up</button>
+        <br/>
+        Min:{minimum}
```

Comment:
I don't think this will work, and I don't think these files for exercise 4 should be in this PR.